e2e/types/manifest.go

```diff
@@ -51,8 +51,10 @@ const (
 type Perturb string
 
 const (
+	// PerturbRestart defines a perturbation that restarts a docker container.
+	PerturbRestart Perturb = "restart"
 	// PerturbStopStart defines a perturbation that stops and then starts a docker container.
-	PerturbStopStart = "stopstart"
+	PerturbStopStart Perturb = "stopstart"
```

Comment:
What is the difference between the above two? To my understanding restart will ALWAYS use the same container and start/stop will potentially use a new container if available, right?
On a side note, what happens during hard VM reset (the process inside of the container doesn't receive a proper signal and a chance to perform a graceful shutdown)?

Reply:
See `e2e/app/perturb.go#perturbService`

Reply:
We need to add some tests for `PurturbKill`